ibis/backends/tests/test_temporal.py

```diff
@@ -401,20 +402,24 @@ def test_temporal_binop_pandas_timedelta(
 @pytest.mark.parametrize(
     'comparison_fn',
     [
-        lambda t: t.timestamp_col > pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col >= pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col < pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col <= pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col == pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col != pd.Timestamp('20100301'),
+        operator.gt,
+        operator.ge,
+        operator.lt,
+        operator.le,
+        operator.eq,
+        operator.ne,
     ],
 )
-@pytest.mark.notimpl(["datafusion", "sqlite"])
```

Comment:
Why is datafusion passing?

Reply:
Could actually be implemented and no need to mark. I need to try it out locally to tell more.